aria/cli/commands/services.py

```diff
@@ -38,6 +41,53 @@ def services():
     pass
 
 
+@services.command(name='show',
+                  short_help='Display service information')
+@aria.argument('service-name')
+@aria.options.verbose()
+@aria.options.show_all
+@aria.options.show_graph
+@aria.options.show_json
+@aria.options.show_yaml
+@aria.pass_model_storage
+@aria.pass_logger
+def show(service_name, model_storage, all, graph, json, yaml, logger):
+    """Show information for a specific service template
+
+    `SERVICE_NAME` is the name of the service to display information on.
+    """
+    logger.info('Showing service {0}...'.format(service_name))
+    service = model_storage.service.get_by_name(service_name)
+
+    if json or yaml:
+        all = True
```

Comment:
i actually didnt mean the flag but only the variables etc.,
although now that i think of it it might also be better to change the flag, as it might be confusing i.e. users might think it will show all service-templates or so..?
in any case what matters to me more is the variables, the flag can still be `all` to the user, but the "destination" of the variable (configurable in `aria.py`) should be renamed.